toolkits/springernature/packages/springernature-submission-header/README.md

```diff
@@ -0,0 +1,62 @@
+# Springer Nature User Details
+
+Springer nature branded 'submission details' component.
+
+## Usage
+
+### HTML
+
+The template requires you have the following data available:
+
+submission: {
+    abstract: {
+        html
+    }
+    articleType,
+    authors: [{
+        givenName,
+        familyName
+    }],
+    journalTitle,
+    revisionVersion,
+    submissionId,
+    submittedTimestamp,
+    title: {
+        html,
+        plainText
+    }
+}
+
+The template included is handlebars, but it is not required. It also assumes you have a `defaultDateFormat` helper available to format your date as `DD Mmm YYYY`, but this need not necessarily be handled in the template.
+
+This component requires the global expander component. It also requires the arrow mixin and the $context--colors var from the brand context (see below).
+
+
+### CSS
+
+Import the styles into your main stylesheet
+
+// core
```

Comment:
do you generate two css files; one for core and one for enhanced? i.e. two http requests? i see you split your component scss in to core.scss and enhanced.scss files which suggests you do generate two css files but this readme makes it sound like you have one main stylesheet that has core and enhanced sections denoted by comments

Reply:
ah yes, bit misleading there